### ISSUE TITLE
"Problem occurred in a component" notifications are annoying

### ISSUE THREAD

**Issue by allanday** [abrt/gnome-abrt#211]
I often get notifications that have the following text:

```
**Oops!**
We're sorry, it looks like a problem occurred in a component...
```
They're really annoying (I might have had something to do with the wording of these notifications, so the blame is likely on me). Reasons for this:

 - There's a lame attempt at humour - it says "Oops!" but there's not much here that's funny.
 - It's apologising, but what for? Often I don't notice the issue except for the notification.
 - It is frustratingly vague. You're interrupting me because it *looks like* there's been a problem?! And then it says *in a component*, which feels deliberately evasive.

A better phrasing might be:

```
**Problem Detected**
A problem has been detected with your system. Click for more information.
```
This sounds helpful and isn't evasive.

**Comment by ernestask** [comment]
I agree that the alternative is better, but why do you say “in a component” is evasive?

**Comment by allanday** [comment]
> but why do you say “in a component” is evasive?

To me it sounds like it knows what the component is, but is refusing to say.
